docs/reference/visibility-modifiers.md

```diff
@@ -9,24 +9,10 @@ title: "Visibility Modifiers"
 
 Classes, objects, interfaces, constructors, functions, properties and their setters can have _visibility modifiers_.
 (Getters always have the same visibility as the property.) 
-There are four visibility modifiers in Kotlin:
+There are four visibility modifiers in Kotlin: `private`, `protected`, `internal` and `public`.
+The default visibility, used if there is no explicit modifier, is `public`.
 
-* `private` --- visible only in the declaring scope and its subscopes (inside the same module);
-* `protected` --- (applicable only to class/interface members) like `private`, but also visible in subclasses;
-* `internal` --- (used by default) visible everywhere within the same module (if the owner of declaring scope is visible);
-* `public` --- visible everywhere (if the owner of declaring scope is visible).
-
-**NOTE**: Functions _with expression bodies_ and all properties declared `public` must always specify return types explicitly. 
-This is required so that we do not accidentally change a type that is a part of a public API by merely
-altering the implementation.
-
-``` kotlin
-public val foo: Int = 5    // explicit return type required
-public fun bar(): Int = 5  // explicit return type required
-public fun bar() {}        // block body: return type is Unit and can't be changed accidentally, so not required
-```
-
-Below please find explanations of these for different type of declaring scopes.  
+Below please find explanations of these for different type of declaring scopes.
```

Comment:
No, actually 'for' is correct here. We have four modifiers for two different types of enclosing scopes. :)